unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PlayerInfoCardHUD/PlayerInfoCardHUDController.cs

```diff
@@ -362,15 +383,18 @@ private void LoadAndShowWearables(UserProfile userProfile)
     {
         wearableCatalogBridge.RequestOwnedWearables(userProfile.userId)
                              .Then(wearables =>
-                             {
-                                 var wearableIds = wearables.Select(x => x.id).ToArray();
-                                 userProfile.SetInventory(wearableIds);
-                                 loadedWearables.AddRange(wearableIds);
-                                 var containedWearables = wearables
-                                     // this makes any sense?
+                              {
+                                  var wearableIds = wearables.Select(x => x.id).ToArray();
```

Comment:
Do you mind converting it to `async`? It is the only legacy method left here.

Reply:
`wearableCatalogBridge.RequestOwnedWearables(userProfile.userId)` its a promise, and i cannot get the result by awaiting it

Reply:
You can `await` `CustomYieldInstruction` but it will not give away the result, still you can fetch it from the saved promise, but you can't catch exceptions in this case.
Well, leave it for now then